tests/unittests/OnnxImporterTest.cpp

```diff
@@ -3172,6 +3172,177 @@ TEST_F(OnnxImporterTest, importEqual) {
   EXPECT_EQ(CMPEQ->getResult().dims()[2], 1);
 }
 
+static void importLogical(const std::string &netFilename,
+                          llvm::ArrayRef<bool> LHS, llvm::ArrayRef<bool> RHS,
+                          llvm::ArrayRef<dim_t> LHSShape,
+                          llvm::ArrayRef<dim_t> RHSShape,
+                          llvm::ArrayRef<dim_t> outputShape,
+                          llvm::ArrayRef<bool> expectedValues) {
+  ExecutionEngine EE{};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+  PlaceholderBindings bindings;
+  Placeholder *graphOutputVar;
+  // Load the .onnxtxt model.
+  Type LHSType(ElemKind::BoolTy, LHSShape);
+  Type RHSType(ElemKind::BoolTy, RHSShape);
+  ONNXModelLoader onnxLD(netFilename, {"LHS", "RHS"}, {&LHSType, &RHSType}, *F);
+  graphOutputVar = EXIT_ON_ERR(onnxLD.getSingleOutput());
+  auto LHSPH = mod.getPlaceholderByNameSlow("LHS");
+  auto *LHSTensor = bindings.allocate(LHSPH);
+  LHSTensor->getHandle<bool>() = LHS;
+  auto RHSPH = mod.getPlaceholderByNameSlow("RHS");
+  auto *RHSTensor = bindings.allocate(RHSPH);
+  RHSTensor->getHandle<bool>() = RHS;
+  EE.compile(CompilationMode::Infer);
+  bindings.allocate(mod.getPlaceholders());
+  EE.run(bindings);
+  auto result = bindings.get(graphOutputVar)->getHandle<bool>();
+  ASSERT_TRUE(result.dims() == (llvm::ArrayRef<dim_t>)outputShape);
+  for (size_t i = 0; i < result.getType().size(); i++) {
+    EXPECT_EQ(result.raw(i), (bool)expectedValues[i]);
+  }
```

Comment:
I guess it's super efficient, but would appreciate there being some line breaks here for logic chunks of code/maybe some simple comments 😄